src/install.js

```diff
@@ -171,6 +173,14 @@ async function main() {
     'impersonationwebhook': { install: argv.iw || argv['impersonationwebhook'], uri: `${fileSource}/ImpersonationWebhook/${filePath}` }
```

Comment:
should you remove it from this resourcesObj?

Reply:
`impersonationwebhook` (https://github.com/razee-io/ImpersonationWebhook) hasn't had a new release in a while, but isn't marked as deprecated and archived the same way as https://github.com/razee-io/encryptedresource.  If the approach for removing deprecated components from existing clusters is approved, adding ImpersonationWebhook to it would likely be a future step.

Reply:
sorry i meant the encrypted resource in this resourcesObj, it just wouldn't let me comment on that line

Reply:
Leaving the ER reference in the `install.js` resourcesObj wouldn't have a _functional_ impact -- the common code shortly after removes any deprecated resources from resourcesObj.  It does help avoid confusion however to remove it, which I have now done 👍